app/src/main/java/org/simple/clinic/summary/bloodsugar/view/BloodSugarSummaryView.kt

```diff
@@ -272,10 +273,12 @@ class BloodSugarSummaryView(
 
     val durationSinceBloodSugarCreated = Duration.between(createdAt, now)
 
-    return if (bloodSugarSummaryConfig.bloodSugarEditFeatureEnabled) {
-      durationSinceBloodSugarCreated <= bloodSugarSummaryConfig.bloodSugarEditableDuration
-    } else {
-      false
+    if (measurement.reading.type !is Unknown) {
```

Comment:
Do you think we can refactor this as well to return `false` when it is `Unknown` else do the rest?
Does that make more sense?